index.js

```diff
@@ -102,9 +102,9 @@ module.exports = function (content) {
     dest: '',
     writeFiles: false,
     formatOptions: config.formatOptions || {
-        ttf: {
-            ts: 0 // it's important to set timestamp to fixed value for ttf generation to have consistent output in long term with the same hash
-        }
+      ttf: {
```

Comment:
Hi

Please open an issue for this and remove your comment from the code so that we can discuss this.

Thx
Jerome